src/app/app.component.html

```diff
@@ -2,8 +2,10 @@
   <div class="spinner"></div>
 </div>
 <ion-app>
-  <ion-split-pane>
-    <ion-menu side="end">
+  <ion-split-pane contentId="main-content">
+    <div id="main-content">
+    <ion-menu side="end" contentId="my-content">
+      <div id="my-content"></div>
```

Comment:
What is the purpose of this change?

Reply:
A warning was being thrown that specified the prior format of the code was deprecated. Therefore, I changed it to match their recommended update.

Reply:
Okay but let's keep these changes in the proper scope. Reformatting/migration changes can be their own issue since it's a cross-cutting concern that affects all future issues/branches.